fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/common/DataDefinitionUtil.java

```diff
@@ -160,6 +160,7 @@ public static void assertValidNames(String... names) {
      * as part of a larger CREATE/ALTER statement. Note: none of the input
      * being dealt with here is external input...so it should be safe already.
      * @param value
+     * @throws IllegalArgumentException if the value contains risky characters
```

Comment:
```suggestion
     * @throws IllegalArgumentException if the value contains unapproved characters
```

Reply:
I don't see anything wrong with saying risky here, especially given that this relates to security